cmd/clean.go

```diff
@@ -3,28 +3,47 @@ package cmd
 import (
 	"bytes"
 	"fmt"
+	"log"
+	"os"
+	"strings"
+
 	"github.com/kubefirst/kubefirst/configs"
+	"github.com/kubefirst/kubefirst/internal/aws"
 	"github.com/kubefirst/kubefirst/internal/reports"
+	"github.com/kubefirst/kubefirst/pkg"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"log"
-	"os"
-	"strings"
 )
 
-// todo delete the s3 buckets associated with the ~/.kubefirst file
 // todo ask for user input to verify deletion?
 // todo ask for user input to verify?
 // cleanCmd removes all kubefirst resources locally for new execution.
 var cleanCmd = &cobra.Command{
 	Use:   "clean",
 	Short: "removes all kubefirst resources locally for new execution",
-	Long: `Kubefirst creates files and folders during installation at your local environment. This command removes and 
-re-create all Kubefirst files.`,
+	Long: `Kubefirst creates files, folders and cloud buckets during installation at your environment. This command removes and 
+re-create all Kubefirst files. To destroy cloud resources you need to specify aditional flags (--destroy-buckets)`,
 	Run: func(cmd *cobra.Command, args []string) {
 
 		config := configs.ReadConfig()
 
+		destroyBuckets, err := cmd.Flags().GetBool("destroy-buckets")
+		if err != nil {
+			log.Panic(err)
```

Comment:
in this case we can use `log.Println()` and call return, this will avoid printing the stacktrace to the console.

Reply:
done